src/libstd/io/mod.rs

```diff
@@ -322,6 +576,17 @@ pub trait Write {
     ///
     /// It is **not** considered an error if the entire buffer could not be
     /// written to this writer.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io::Write;
+    ///
+    /// let mut buffer = Vec::new();
```

Comment:
Ah I'm not too worried about functionality here, so I'm fine considering them all `no_run`. I feel like files may be a bit better here as it's probably similar to what you're using in the general case. For example `Vec` has extra stuff like `BufRead` implemented on it, but `File` is the "bare bones" reader (minus seek I guess)